security/security-spring/src/main/resources/applicationContext-security.xml

```diff
@@ -622,7 +622,7 @@
         <authentication-manager alias="authenticationManager"/>
     </b:beans>
 
-    <b:beans profile="social_auth">
+    <b:beans profile="social_auth_google | social_auth_microsoft">
```

Comment:
I never tried using pipe here.  Have you verified this works?

Reply:
it working. tested